cmd/commands/monitor/command_monitor.go

```diff
@@ -2,6 +2,7 @@ package monitor
 
 import (
 	"errors"
+	node_cmd "github.com/mysterium/node/cmd"
```

Comment:
why cmd, wrong package name in this scope?

Reply:
Yes - all command methods uses `cmd` for instance variable. Didn't want to rename all instance variables, because package name `cmd` is not the best one and we should rename this package instead of trying to avoid using `cmd` anywhere.